cips/versioned_gas_scheduler_variables.md

```diff
@@ -0,0 +1,60 @@
+| cip | 24 |
+| - | - |
+| title | Versioned Gas Scheduler Variables |
+| description | Transition to hard fork-only updates for gas scheduler variables |
+| author | Nina Barbakadze (@ninabarbakadze) |
+| discussions-to |  |
+| status | Draft |
+| type | Standards Track |
+| category | Core |
+| created | 2024-07-24 |
+
+## Abstract
+
+Introduce versioning for on-chain governance modifiable gas scheduler parameters, such as `blob.GasPerBlobByte` and `auth.TxSizeCostPerByte`, to only allow changes during hard fork upgrades.
```

Comment:
You don't have to use "such as" when these two variables are the exhaustive list.

Gas scheduler parameters: `blob.GasPerBlobByte` and `auth.TxSizeCostPerByte` will no longer be modifiable by governance but may only change via a hard fork upgrade